services/bettersocial/user/BetterSocialUnblockUserV2.js

```diff
@@ -43,6 +69,8 @@ const BetterSocialUnblockUserV2 = async (token, selfUserId, targetUserId) => {
   }
 
   BetterSocialScoreUnblockUser(selfUserId, targetUserId);
+  if (anonymousBlockFromSign)
+    BetterSocialScoreUnblockUser(selfUserId, targetAnonymousUserId?.user_id);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

This code updates the social score for both the target user and the anonymous user upon unblocking. The logic seems correct, but it would be better to check if `targetAnonymousUserId` is not null before calling `BetterSocialScoreUnblockUser`. This way, you can avoid unnecessary function calls when there's no anonymous user.
```diff
- BetterSocialScoreUnblockUser(selfUserId, targetAnonymousUserId?.user_id);
+ if(targetAnonymousUserId) {
+   BetterSocialScoreUnblockUser(selfUserId, targetAnonymousUserId.user_id);
+ }
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->